kubernetes/cmsweb/services/das-server.yaml

```diff
@@ -42,6 +42,15 @@ spec:
         env:
         - name: X509_USER_PROXY
           value: /etc/proxy/proxy
+        livenessProbe:
```

Comment:
@muhammadimranfarooqi it seems to me you duplicated livenessProbe in das-server which I added before and now it appears now twice. Pease check and fix.